TestFoundation/TestProcessInfo.swift

```diff
@@ -104,7 +107,12 @@ class TestProcessInfo : XCTestCase {
         XCTAssertNil(env["bad3"])
         XCTAssertNil(env["bad3="])
 
+#if os(Windows)
+        // On Windows, adding an empty environment variable removes it from the environment
+        XCTAssertNil(env["var1"])
```

Comment:
Could we not emulate this in the subscript operator?  Setting it to nothing would set it to the empty string.

Reply:
As far as I'm aware, you can't have an environment variable set to `""` or be unset on Windows. Doing so simply removes it from the environment. Since it's not in the environment, I think it makes sense to have it return nil there.

Reply:
Bleh, yeah, `putenv` behaves like set.  Okay, that seems reasonable.